specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,235 @@
+    fn sbt_supply_for_owner(&self, account: AccountId);
+
+    /// Query for sbt tokens
+    fn sbt_tokens(&self, from_index: Option<U64>, limit: Option<u32>) -> Vec<Token>;
+
+    /// Query sbt tokens by owner
+    fn sbt_tokens_for_owner(
+        &self,
+        account: AccountId,
+        from_index: Option<U64>,
+        limit: Option<u32>,
+    ) -> Vec<Token>;
+
+    /// Optional. If the SBT implementaiton assures that one account can have maximum one SBT
+    /// the the following function should be implemented.
+    /// Returns Some(Token) if an `account` owns an SBT, otherwise returns None.
+    fn sbt_token_for_owner(&self, account: AccountId) -> Option<Token> {}
+
+    /**********
+    * TRANSACTIONS
+    **********/
+
+    /// creates a new, unique token and assigns it to the `receiver`.
+    #[payable]
+    fn sbt_mint(&mut self, metadata: TokenMetadata, receiver: AccountId);
+
+    /// sbt_recover reassigns all tokens from the old owner to a new owner,
+    /// and registers `old_owner` to a burned addresses registry.
+    /// Must be called by operator.
+    /// Must provide 5 miliNEAR to cover registry storage cost. Operator should
+    ///   put that cost to the requester (old_owner), eg by asking operation fee.
+    #[payable]
+    fn sbt_recover(&mut self, from: AccountId, to: AccountId);
+
+    /// sbt_renew will update the expire time of provided tokens.
+    /// `expires_at` is a unix timestamp (in seconds).
+    #[payable]
+    pub fn sbt_renew(&mut self, tokens: Vec<TokenId>, expires_at: u64, memo: Option<String>);
+}
+```
+
+### Logs
+
+```typescript
+interface SbtEventLogData {
+  standard: "nepXXX";
+  version: "1.0.0";
+  event: "sbt_mint" | "sbt_recover" | "sbt_renew";
+  data: SbtMintLog[] | SbtRecoverLog[] | SbtRenew[];
+}
+
+// An event emitted when a new SBT is minted.
+// Arguments
+// * `owner`: "account.near"
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtMintLog {
+  owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when existing SBTs are revoked and burned.
+// Arguments
+// * `owner`: "account.near"
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRevokeLog {
+  owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when a recovery process succeeded to reassign SBT.
+// Arguments
+// * `old_owner`: "old_account.near"
+// * `new_owner`: "new_account.near"
+// * `token_ids`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRecoverLog {
+  old_owner: string;
+  new_owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when a existing tokens are renewed.
+// Arguments
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRenewLog {
+  tokens: uint64[];
+  memo?: string;
+}
+```
+
+Whenever a recovery is made in a way that an existing SBT is burned, the `SbtRevokeLog` event must be emitted.
+
+```typescript
```

Comment:
Is this part of the contract interface? Sry I am not versed in AS contracts, and I would find it odd to define the standard in two different SDKs. If this is part of the contract interface, shouldn't we prefix the methods with `sbt_`?

In general, there is no description as to what this interface description is for.